app/models/work_search_cleanser.rb

```diff
@@ -0,0 +1,129 @@
+class WorkSearchCleanser
+
+  # For various reasons, some options come in needing processing/cleanup
+  # before we use them for searching. May be indicative of code that needs
+  # cleaning up elsewhere in the app.
+  def self.clean(options)
+    new(options).clean
+  end
+
+  attr_accessor :options
+
+  def initialize(options)
+    @options = options || {}
+  end
+
+  def clean
+    set_parent_fields!
+    set_tag_fields!
+    set_sorting!
+    set_language!
+    clean_up_angle_brackets
+
+    # Clean up blank options from forms
+    options.delete_if { |key, value| value.blank? }
+  end
+
+  private
+
+  def set_parent_fields!
```

Comment:
We have similar code in WorkQuery:

https://github.com/otwcode/otwarchive/blob/f91b3606fd270add0933d16049ef49ebe244551c/app/models/search/work_query.rb#L42-L57

The WorkQuery version adds only the user ID if the parent is a user, instead of all the pseud IDs of that user, which should make loading orphan_account's works faster ([AO3-4871](https://otwarchive.atlassian.net/browse/AO3-4871)).

Can we remove `set_parent_fields!`?

Reply:
Ideally, we'd get rid of the whole class, but I thought we might want to take things step by step. I can take a look and see if that causes any problems.